### PR TITLE
Extend TaskQueue for dbus tasks

### PR DESCRIPTION
**Pull request by M4rtinK**
Extend the TaskQueue API to make adding DBus Tasks easy.

### CODE REVIEW

**Comment by M4rtinK** [comment]
NOTE: This should be merged after #2073 and #2077 are merged, so the new tasks they add can use this API as well.

File: pyanaconda/installation_tasks.py [hunk 1]
```diff
@@ -321,6 +322,17 @@ def append(self, item):
         item.parent = self
         self._list.append(item)
 
+    @synchronized
+    def append_dbus_tasks(self, module, dbus_tasks):
+        """Append DBus Tasks from a module to the TaskQueue.
+
+        :param module: DBusServiceIdentifier instance corresponding to an Anaconda DBus module
```
Comment (poncovka) [inline pyanaconda/installation_tasks.py]: I would rename it to `service_id`, so it cannot be confused with a proxy of a DBus module.

Comment (M4rtinK) [reply pyanaconda/installation_tasks.py]: Sounds good, changed. :)

File: pyanaconda/installation.py [hunk 2]
```diff
@@ -90,34 +89,28 @@ def _prepare_configuration(storage, payload, ksdata):
     os_config = TaskQueue("Installed system configuration", N_("Configuring installed system"))
     os_config.append(Task("Configure authselect", ksdata.authselect.execute))
 
+    # add installation tasks for the Security DBus module
```
Comment (jkonecny12) [inline pyanaconda/installation.py]: Please remove these comments. By this comment you are telling the same as you can easily read from the code. If you want to have a comment here, then merge them together to ``add installation tasks from DBus modules``.

Comment (M4rtinK) [reply pyanaconda/installation.py]: The order of the modules is sometimes important & it simply seems more readable this way.

Comment (jkonecny12) [reply pyanaconda/installation.py]: Both ways are good to me. One is just slightly better.
Still you have my ACK.

**Comment by M4rtinK** [comment]
jenkins, test this please

**Pull request merged**